lib/containers/k8s.pm

```diff
@@ -92,4 +112,47 @@ sub install_oc {
     assert_script_run('mv oc /usr/local/bin');
     assert_script_run('oc');
 }
+
+=head2 apply_manifest
+Apply a kubernetes manifest
+=cut
+
+sub apply_manifest {
+    my ($manifest) = @_;
+
+    my $path = sprintf('/tmp/%s.yml', random_string(32));
+
+    script_output("echo -e '$manifest' > $path");
+    upload_logs($path, failok => 1);
+
+    assert_script_run("kubectl apply -f $path");
+}
+
+=head2 find_pods
+Find pods using kubectl queries
+=cut
+
+sub wait_for_job_complete {
```

Comment:
I guess, it would be less ambiguous to express that the function waits for a `kube` job/pod to finished

Reply:
good point, but it is inside k8s lib, is necessary to specify? wait_for_k8s_job_complete, for instance?

Reply:
I think that it would be handy, mainly for people that are often `greping` test repo for function names as me. :D

Reply:
Fixed then